blocks/contact-us/contact-us.css

```diff
@@ -0,0 +1,168 @@
+.contact-us-container .contact-us-wrapper {
+    max-width: unset;
+    padding: 0;
+    background-color: var(--footer-background-color);
+}
+
+.contact-us.block {
+    --button-background-color: #0f61e6;
+    --border-color: hsl( 0deg 0% 100% / 20%);
+    --font-familly: 'Open Sans', arial, sans-serif;
+
+    max-width: 90%;
```

Comment:
Double check this at the different breakpoints. I don't think it's entirely accurate or this level of consistent across all of them.

Also, if you look at the live site at 900px, you can see that there is more space between the content and the borders of the boxes.